### PR TITLE
Add three functions for contents searching in VM

### PR DESCRIPTION
**Pull request by xiaodwan**
1) Rename get_vm_xorg to vm_xorg_search, use 'grep' instead of
   'cat' to decrease unnecessary log printing
2) Add vm_journal_search for journal log checking
3) Add vm_general_search for general cmd output checking

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.convert_from_file.positive_test.linux.input_mode.ova.aws.ec2.output_mode.rhev.rhv_upload: PASS (6211.67 s)
(1/1) type_specific.io-github-autotest-libvirt.convert_from_file.positive_test.linux.input_mode.ova.normal.rhel6_5.output_mode.rhev.rhv_upload: PASS (5742.94 s)

**Comment by xiaodwan** [comment]
@chunfuwen @chloerh Please help to review this PR, thanks.

File: virttest/utils_v2v.py [hunk 1]
```diff
@@ -618,16 +618,109 @@ def wait_for_x_start(self, timeout=30):
...
+        Search journalctl log on vm
+
+        :param substr: The expected string.
+        :param options: the journalctl options
+        :param flags: A RegexFlag, Please refer RE moudule of python
+        :return: True if search result meets expectation, otherwise False
+        """
+        cmd = "journalctl"
+        if options:
+            cmd = "journalctl %s" % options
```
Comment (chloerh) [inline virttest/utils_v2v.py]: Why not `cmd += str(options)`

File: virttest/utils_v2v.py [hunk 2]
```diff
@@ -618,16 +618,109 @@ def wait_for_x_start(self, timeout=30):
...
+        :param cmd: A command to be excuted
+        :param substr: The expected string in output of cmd.
+        :param flags: A RegexFlag, Please refer RE moudule of python
+        :param ignore_status: If True, will not check command return status.
+        :param debug: If True, will print cmd output.
+        :return: True if search result meets expectation, otherwise False
+        """
+        status, output = self.run_cmd(cmd, debug)
+
+        if ignore_status and re.search(substr, output, flags):
```
Comment (chunfuwen) [inline virttest/utils_v2v.py]: combine together by if re.search(substr, output, flags) and (ignore_status or status == 0): return True ?

File: virttest/utils_v2v.py [hunk 3]
```diff
@@ -618,16 +618,109 @@ def wait_for_x_start(self, timeout=30):
...
+            logging.debug("Get xorg logs shell script:\n%s", get_xorg_logs)
+
+            # The first element is a malformed get_xorg_logs string, it
+            # should be removed.
+            xorg_files = self.run_cmd(get_xorg_logs, debug=False)[
+                1].strip().splitlines()
+            if len(xorg_files) > 0:
+                xorg_file_list.extend(xorg_files[1:])
+        else:
+            logging.debug("Get UID_MIN failed: %s", uid_min)
```
Comment (chunfuwen) [inline virttest/utils_v2v.py]: do you need return False directly here?

Comment (xiaodwan) [reply virttest/utils_v2v.py]: No, It should continue to check all general possible xorg files.

File: virttest/utils_v2v.py (hunk 2 above)
Comment (xiaodwan) [reply virttest/utils_v2v.py]: Thanks, Done

File: virttest/utils_v2v.py (hunk 1 above)
Comment (xiaodwan) [reply virttest/utils_v2v.py]: Thanks, Done

**Pull request merged**